CHANGES/6223.bugfix

```diff
@@ -0,0 +1 @@
+Including file type extension when uploading collections
```

Comment:
Can this also indicate that it fixes the user's busted database with a data migration.